src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/OrganisationEntityResponse.java

```diff
@@ -49,10 +49,14 @@ private void getOrganisationEntityResponse(Organisation organisation, Boolean is
         if (!organisation.getUsers().isEmpty()) {
             this.superUser = new SuperUserResponse(organisation.getUsers().get(0));
         }
-        this.paymentAccount = organisation.getPaymentAccounts()
-                .stream()
-                .map(pbaAccount -> new PbaAccountResponse(pbaAccount).getPbaNumber())
-                .collect(toList());
+        if (organisation.getPaymentAccounts().size() > 0) {

```

Comment:
does this mean Stream doesn't check size internally ? as per me stream() always checks size and move ahead, don't need to check size manually.